crates/bevy_asset/src/lib.rs

```diff
@@ -53,7 +53,7 @@ impl Default for AssetServerSettings {
     fn default() -> Self {
         Self {
             asset_folder: "assets".to_string(),
-            watch_for_changes: false,
+            watch_for_changes: cfg!(feature = "bevy_shader_hot_reloading"),
```

Comment:
Just noting that I did this for convenience. It saves adding this as a resource in example just to set it to true when hacking/testing stuff.